packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -27,49 +29,94 @@ type Props = {
      * clicks too.)
      */
     onClickCloseButton: () => void,
-};
 
-/**
- * A two-column modal layout.
- */
-export default class TwoColumnModal extends React.Component<Props> {
+    /** The footer to show under the main column's content. */
+    footer?: React.Node,
+
+    /** The main column's content. */
+    content: React.Node,
+
+    /** Should the sidebar be made smaller? (e.g. 1 column wide) */
```

Comment:
nit: should this be an "i.e." ("in other words") instead of an "e.g." ("for example")? Or are we maybe using that to suggest that being exactly 1 column isn't part of the contract, so we're framing it as an example of what we might do?

Reply:
Right - I'll fix this, I think it should probably just be `(exactly 1 column wide)`